commandapi-core/src/main/java/dev/jorel/commandapi/arguments/AbstractArgument.java

```diff
@@ -224,6 +225,13 @@ public final Impl withRequirement(Predicate<CommandSender> requirement) {
 		return instance();
 	}
 
+	/**
+	 * Resets the requirements for this command
+	 */
+	public final void resetRequirements() {
+		this.requirements = s -> true;
+	}
+
```

Comment:
Should this method return an instance of the Argument? It kinda feels like a method someone might use while building an argument, though on the other hand, they should probably just give it the requirements they want and not bother resetting them. 

You did bother making this a separate public method when it could have been directly in `copyPermissionsAndRequirements` or private. If this is a method external developers should use, it'd be nice to put in in the docs.

I don't know what's right for this method though, depends on your vision for it.